lib/features/email/presentation/email_view.dart

```diff
@@ -441,20 +440,20 @@ class EmailView extends GetWidget<SingleEmailController> {
       return const SizedBox.shrink();
     }
     return Obx(() {
-      if (controller.emailContents.isNotEmpty) {
-        final allEmailContents = controller.emailContents.asHtmlString;
+      if (controller.emailContents.value?.isNotEmpty ?? false) {
```

Comment:
IMO, it should only be checked if it is different from `null` because there may be cases where the content is empty